### PR TITLE
Patch Release: update team process documentation

### PR DESCRIPTION
**Pull request by tpepper**
Add description of how we are working together as a team now to track the evolving state of cherry pick PRs and arrive at consensus on whether or not to approve each for merge.

### CODE REVIEW

**Comment by tpepper** [comment]
@kubernetes/patch-release-team FYI

Also this should represent closure on #369

File: release-engineering/role-handbooks/patch-release-team.md [hunk 1]
```diff
@@ -256,6 +263,174 @@ For each cherry-pick request:
...
+The spreadsheets run in reverse chronological order with the newest
+patch release's activity at the top.
+
+Sections are made per patch release.
+
+A row is entered in a section per cherry pick pending ahead of that
+patch release.  Within each of these sections, cherry picks are
+listed in reverse PR number order, newest on top.  To add a notation
+for a newly inbound cherry pick, add a row above the current newest
+row.
```
Comment (idealhack) [inline release-engineering/role-handbooks/patch-release-team.md]: how often do pst members enter or modify new items?

File: release-engineering/role-handbooks/patch-release-team.md [hunk 2]
```diff
@@ -256,6 +263,174 @@ For each cherry-pick request:
...
+for most PRs).  The parent PRs may have poor labelling resulting
+in incomplete labels on the cherry pick.  Attempt to clean then up
+if possible or comment asking for owners to better disposition them.
+When recording unsure status from labels add a "?" in the spreadsheet
+cell to indicate the uncertainty.
+
+### Future Work
+
+ * It would be awesome to replace this manual stateful review workflow with a
+   tool.  Such a tool appears to need invented.
```
Comment (idealhack) [inline release-engineering/role-handbooks/patch-release-team.md]: what are some features or specs for such a tool? if it's needed very much, let's start to think about it.

File: release-engineering/role-handbooks/patch-release-team.md (hunk 1 above)
Comment (hoegaarden) [reply release-engineering/role-handbooks/patch-release-team.md]: Depends. I guess less often, after we cut a patch, more often just before we cut a patch. I think I had times where I did check on cherry pick PR only once a week and I had times where I checked and updated cherry pick PRs more then once a day. Not sure how other's on the team handle that though.

File: release-engineering/role-handbooks/patch-release-team.md (hunk 2 above)
Comment (tpepper) [reply release-engineering/role-handbooks/patch-release-team.md]: No spec exists, but automation in support of the manual inspect, triage, share thoughts, discuss, form consensus would be useful.  As I write that...It almost sounds like Gerrit.  /me ducks from the thrown tomatoes.

File: release-engineering/role-handbooks/patch-release-team.md (hunk 1 above)
Comment (tpepper) [reply release-engineering/role-handbooks/patch-release-team.md]: Over in https://github.com/kubernetes/community/pull/4081 I note we scrub things about weekly, but more often just before release.

**Pull request merged**